pages/linux/users.md

```diff
@@ -0,0 +1,11 @@
+# users
+
+> Display a list of logged in users.
```

Comment:
How about ..
`Lists the login names of the users currently on the system, in sorted order`

Reply:
I think "logged in users" is fine.

Reply:
I think that's a bit too verbose, @mfrw. If possible, we prefer more concise descriptions as opposed to longer ones as they're easier to understand quickly :-)